02-linked-lists/02-linked-lists-implementation.md

```diff
@@ -1215,7 +1226,8 @@ At each iteration:
 - Redirect both `previous` and `current` to point to their respective next nodes in the list
 
 Still feeling stuck? Check this video walkthrough of the solution.
-<!-- TODO: Add video walkthrough to explanation -->
+
+<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?id=cb846900-4327-4727-833d-aef601438473&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&captions=true&interactivity=all" height="360" width="640" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
```

Comment:
Because a previous pointers exist in doubly linked lists, could be helpful to highlight that `previous` here is not a Node attribute but a variable we are using to track where we are (or where we were) in the list.

 I feel like this one because it's so complicated, it might be helpful to have the diagram shown alongside the code in a split screen. But it might also be hard to read both the code and diagram that way?